History.md

```diff
@@ -24,6 +24,7 @@ This is the first Express 5.0 alpha release, based off 4.10.1.
     - `req.acceptsEncoding` - use `req.acceptsEncodings`
     - `req.acceptsLanguage` - use `req.acceptsLanguages`
     - `res.json(obj, status)` signature - use `res.json(status, obj)`
```

Comment:
No, because that is the change log for the first alpha. You can't changed what has already been released.